WebService/Properties/launchSettings.json

```diff
@@ -10,6 +10,8 @@
         "PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING": "your DocumentDb connection string",
         "PCS_AZURE_STORAGE_ACCOUNT": "your Azure Storage Account connection string",
         "PCS_STORAGEADAPTER_WEBSERVICE_URL": "http://localhost:9022/v1",
+        "PCS_RESOURCE_GROUP_LOCATION": "your Azure resource group location",
+        "PCS_VMSS_NAME": "your vm scale set name",
```

Comment:
please update also `/.vscode/launch.json` for VSCode users, and `/WebService/WebService.csproj` for VS-for-Mac users

Reply:
done